doc/tutorials/using_hybrid_mode_on_nuc.rst

```diff
@@ -10,13 +10,13 @@ launched by a Device Model in the Service VM.
 .. figure:: images/ACRN-Hybrid.png
```

Comment:
We should update this image based on what's on this page: https://projectacrn.github.io/latest/introduction/index.html (also because it mentions safety at the moment).

@dbkinder , do you know who has the original versions of these pictures?

Reply:
Where do you see `/root/yaag`?

Reply:
> Where do you see `/root/yaag`?

At the very bottom of the updated configurator-postvm.png from @jackwhich 
![image](https://user-images.githubusercontent.com/11063618/175646671-84684ce1-349d-44c5-a0bd-205af91ca7fe.png)

Reply:
After the PR is merged, I can offer to update both images: (1) configurator-postvm.png to align with the main GSG flow, and (2) scenario diagram in hybrid GSG to match the format of image in What Is ACRN.